masterlist.yaml

```diff
@@ -7722,6 +7722,75 @@ plugins:
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("WhiterunMarket.esp") and not active("AncientLand-DarksWhiterunMarket.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Whiterun Valley - Hold Expansion](https://www.nexusmods.com/skyrimspecialedition/mods/11917)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("WhiterunValley3.esp") and not active("AncientLand-WhiterunValley.esp.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Enhanced Solitude Docks](https://www.nexusmods.com/skyrimspecialedition/mods/28349)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("Enhanced Solitude Docks.esp") and not active("AncientLand-EnhancedSolitudeDocks.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Oakwood](https://www.afkmods.com/index.php?/files/file/2178-oakwood)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("Oakwood.esp") and not active("AncientLand-Oakwood.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Schlitzohr''s Oakwood](https://www.nexusmods.com/skyrimspecialedition/mods/61007)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("AnotherOakwood.esp") and not active("AncientLand-AnotherOakwood.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Pirates of The North SSE](https://www.nexusmods.com/skyrimspecialedition/mods/28985)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("PiratesNorth.esp") and not active("AncientLand-PiratesOfTheNorth.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Camp Argentum SSE](https://www.nexusmods.com/skyrimspecialedition/mods/14226)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("CampArgentum.esp") and not active("AncientLand-CampArgentum.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Beginner''s Shack in Riverwood](https://www.nexusmods.com/skyrimspecialedition/mods/6238)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("Eli_RiverwoodShack.esp") and not active("AncientLand-EliRiverwoodShack.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Leaps of Faith - A Misc Quest](https://www.nexusmods.com/skyrimspecialedition/mods/53074)'
+          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140)'
+        condition: 'active("LeapsOfFaith.esp") and not active("AncientLand-LeapsOfFaith.esp")'
```

Comment:
The `clean` key should be sorted after the `msg` key, & the `patch3rdParty` messages should be sorted alphanumerically by their subs. Also, since users will already have the mods being patched, links for them aren't necessary. There's also an extra `.esp` in the filename of `AncientLand-WhiterunValley.esp`:

```suggestion
    msg:
      - <<: *patch3rdParty
        subs:
          - 'Arthmoor''s Oakwood'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("Oakwood.esp") and not active("AncientLand-Oakwood.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Beginner''s Shack in Riverwood'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("Eli_RiverwoodShack.esp") and not active("AncientLand-EliRiverwoodShack.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Camp Argentum'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("CampArgentum.esp") and not active("AncientLand-CampArgentum.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Dark Forests of Skyrim'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("Dark_Forests_FULL.esp") and not active("AncientLand-DarkForests.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Dark''s Whiterun market'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("WhiterunMarket.esp") and not active("AncientLand-DarksWhiterunMarket.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Enhanced Landscapes'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("Enhanced Landscapes.esp") and not active("AncientLand-EnhancedLandscapes.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Enhanced Solitude Docks'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("Enhanced Solitude Docks.esp") and not active("AncientLand-EnhancedSolitudeDocks.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Lanterns Of Skyrim II'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("Lanterns Of Skyrim II.esm") and not active("AncientLand-LoS.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Leaps of Faith - A Misc Quest'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("LeapsOfFaith.esp") and not active("AncientLand-LeapsOfFaith.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Pirates of The North'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("PiratesNorth.esp") and not active("AncientLand-PiratesOfTheNorth.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Schlitzohr''s Oakwood'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("AnotherOakwood.esp") and not active("AncientLand-AnotherOakwood.esp")'
      - <<: *patch3rdParty
        subs:
          - 'Whiterun Valley - Hold Expansion'
          - '[Ancient Land Patches](https://www.nexusmods.com/skyrimspecialedition/mods/54140/)'
        condition: 'active("WhiterunValley3.esp") and not active("AncientLand-WhiterunValley.esp")'
    clean:
      - crc: 0xB5FDA10D
        util: 'SSEEdit v4.0.4'
```